distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -6,130 +6,149 @@
 :order: 01
 
 Introduction: ${catalog-ui} represents the most advanced search interface available with ${branding}.
-It provides a metadata search and discovery, resource retrieval, and workspace management with a 3d or optional 2d map visualization.
+It provides metadata search and discovery, resource retrieval, and workspace management with a 3d or optional 2d map visualization.
```

Comment:
nit: 3D and 2D are capitalized elsewhere.

Reply:
fixed